rotkehlchen/tests/api/test_misc.py

```diff
@@ -521,26 +521,28 @@ def test_cache_deletion(rotkehlchen_api_server):
 
 
 @pytest.mark.parametrize('have_decoders', [True])
-def test_events_mappings(rotkehlchen_api_server):
+@pytest.mark.parametrize('added_exchanges', [(Location.KRAKEN, Location.BINANCE)])
+def test_events_mappings(rotkehlchen_api_server_with_exchanges):
     """
     Test different mappings and information that we provide for rendering events information
     - Test that the structure for types mappings is correctly generated
     - Test that the valid locations are correctly provided to the frontend
     """
     response = requests.get(
         api_url_for(
-            rotkehlchen_api_server,
+            rotkehlchen_api_server_with_exchanges,
             'typesmappingsresource',
         ),
     )
     result = assert_proper_response_with_result(response)
     assert 'per_protocol_mappings' in result
     assert 'global_mappings' in result
     assert 'event_category_details' in result
+    assert 'kraken' in result['exchange_mappings'] and len(result['exchange_mappings']) == 1
```

Comment:
This could be checked in a single condition with `result['exchange_mappings'].keys() == {'kraken'}`